api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
+	Recent   struct {
+		Problem   string `json:"problem"`
+		Submitted string `json:"submitted_at"`
+	}
+	Fetched []string `json:"fetched"`
+	Skipped []string `json:"skipped"`
```

Comment:
I think `FetchedProblems` and `SkippedProblems` are good and descriptive names here. I'll go with that!